api/spec/controllers/mno_enterprise/jpi/v1/app_instances_sync_controller_spec.rb

```diff
@@ -77,18 +77,18 @@ module MnoEnterprise
     end
 
     describe "POST #create" do
-      xit "to spec: cannot stub 'post /app_instances_syncs data%5Bmode%5D=a_mode'"
+      it "to spec: cannot stub 'post /app_instances_syncs data%5Bmode%5D=a_mode'"
```

Comment:
I left it to still get a pending warning as the spec below is barely testing anything (just the authorization)